modules/docs/src/main/tut/migration.md

```diff
@@ -28,12 +28,13 @@ Doobie imports have been split into normal and implicit slices, to mirror the pa
 
 ## Data Types
 
-The IO data types provided by fs2 and doobie have been subsumed by `cats.effect.IO` or any data type with a `cats.effect.Aync` instance, such as `monix.eval.Task`.
+The IO data types provided by fs2 and doobie have been subsumed by `cats.effect.IO` or any data type with a `cats.effect.Async` instance, such as `monix.eval.Task`.
 
 | 0.4.x Data Type      | 0.5.x Data Type  |
 |----------------------|------------------|
 | `fs2.Task`           | `cats.effect.IO` |
 | `doobie.util.IOLite` | `cats.effect.IO` |
+| `monix.eval.Task`    | `cats.effect.IO` |
```

Comment:
This table row contradicts with what's written above the table :) If someone was using `monix.eval.Task`, it doesn't need to be changed to `cats.effect.IO`.